docs/examples/viz_markers.py

```diff
@@ -0,0 +1,52 @@
+"""
+======================================================================
+Fury Markers
+======================================================================
+
+This it's a example which shows how to use the marker actor.
+"""
+import numpy as np
+import vtk
+import vtk.util.numpy_support as vtknp
+from fury import actor, window, colormap as cmap
+n = 10000
+
+"""
+The are nine types 2d markers: circle, square, diamond,
+triangle, pentagon, hexagon, heptagon, cross and plus.
+"""
+marker_symbols = ['o', 's', 'd', '^', 'p', 'h', 's6', 'x', '+']
+markers = [
+    np.random.choice(marker_symbols)
+    for i in range(n)]
+
+centers = np.random.normal(size=(n, 3), scale=10)
+
+colors = np.random.uniform(size=(n, 3))
+# In addition, as similar to networkx,  you can control the
+# edge color and edge width for each marker
```

Comment:
Same comment as above. No need to precise networkx

Reply:
Ok